indico/modules/rb/__init__.py

```diff
@@ -70,14 +72,16 @@ def _extend_admin_menu(sender, **kwargs):
 @signals.menu.items.connect_via('top-menu')
 def _topmenu_items(sender, **kwargs):
     if config.ENABLE_ROOMBOOKING:
-        yield TopMenuItem('room_booking', _('Room booking'), url_for('rb.roombooking'), 80)
+        if not rb_settings.get('hidden_from_unauthorised_users') or rb_check_user_access(session.user):
+            yield TopMenuItem('room_booking', _('Room booking'), url_for('rb.roombooking'), 80)
 
 
 @signals.menu.items.connect_via('event-management-sidemenu')
 def _sidemenu_items(sender, event, **kwargs):
     if config.ENABLE_ROOMBOOKING and event.can_manage(session.user):
-        yield SideMenuItem('room_booking', _('Room booking'), url_for('rb.event_booking_list', event), 50,
-                           icon='location')
+        if not rb_settings.get('hidden_from_unauthorised_users') or rb_check_user_access(session.user):
```

Comment:
I think the page should not be hidden regardless of room booking access - only the UI to proceed to booking a room should be hidden since it makes perfect sense for any event organizer to see which rooms have been booked for the event.

Reply:
Okay

Reply:
Hi @ThiefMaster, this page already has a check access and gives `Forbidden(_('You are not authorized to access the room booking system.'))` for unauthorised users so I think it's okay to hide that link as well since the UI is already unaccessible.